src/packages/sdk/src/__tests__/getGenerators/getGenerators.test.ts

```diff
@@ -1,9 +1,9 @@
 import path from 'path'
 import { getGenerators } from '../../getGenerators'
-import { pick } from '../../pick'
 import { omit } from '../../omit'
+import { pick } from '../../pick'
 
-jest.setTimeout(10000)
+jest.setTimeout(20000)
```

Comment:
It seems that 10 seconds was not long enough. Example: https://github.com/prisma/prisma/runs/1226432839#step:6:600. So trying 20 seconds...